eventuals/static-thread-pool.h

```diff
@@ -42,7 +63,8 @@ class StaticThreadPool : public Scheduler {
       : Requirements(std::string(name), std::move(pinned)) {}
 
     Requirements(std::string name)
-      : name(std::move(name)) {}
+      : name(std::move(name)),
+        pinned(Pinned::Any()) {}
 
     Requirements(std::string name, Pinned pinned)
```

Comment:
Let's delete the constructor that only takes `name` and make the constructor that takes `Pinned` have a default value of `Pinned::Any()`.